### ISSUE TITLE
[Enhancement]  [FilePicker] [Android] Allow User to Pick files from Google Drive/Photos

### ISSUE THREAD

**Issue by devendra4916** [xamarin/Essentials#1497]
Currently, Google Drive ☁️ option is not available to pick files from device.
To allow other options like google drive/google photos etc.

could you please update code as:
Essentials/Xamarin.Essentials/FilePicker/FilePicker.android.cs
```
intent.SetAction(Intent.ActionGetContent); //This will additionally shows contact option
intent.AddCategory(Intent.CategoryOpenable); // to avoid other options like Contacts

//Additionally, you can allow user to choose local files only
intent.PutExtra(Intent.ExtraLocalOnly, allowLocalOnly);
```

Thanks in Advance.. 🙏 🥳

**Comment by mattleibow** [comment]
When I use the FilePicker, I get options to pick files from local, Google Drive and OneDrive.

Do you have any more info to share about the scenario? Like the OS version, the device, what is the code you are using?

**Comment by mattleibow** [comment]
Closing this as I think this was fixed in #1555
